src/ui/camera.js

```diff
@@ -545,6 +545,7 @@ class Camera extends Evented {
      * Returns a {@link CameraOptions} object for the highest zoom level
      * up to and including `Map#getMaxZoom()` that fits the bounds
      * in the viewport at the specified bearing.
+     * This API isn't supported with globe projection.
```

Comment:
Nit: Clarity of language to make it clear that other functions are supported

```suggestion
     * This function isn't supported with globe projection.
```